### DISCUSSION TITLE
Debug "Show Options" color

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#5273] [Ideas]
Here is a nit for you.... In the Debug "show option" page could the selection box color match the "box displayed" in the image'
Maybe the image will make my explanation clearer as to what I mean.
![Zones](https://user-images.githubusercontent.com/59022856/215194014-b95d08aa-471c-4b97-8802-4593036ce125.jpg)

**Comment by NickM-27** [comment]
That would work for motion boxes and regions but bounding boxes have different colors based on the type of object and zones will have different colors when there are multiple zones.

**Reply by 18923** [reply]
Seemed like a good idea as I often forget what each frame means.
Awesome work you are all doing. For me this is just a hobby to keep my mind active, especially in the winter months.

**Comment by leccelecce** [comment]
I would second the need for some kind of explanation, as a newcomer to the debug feature I'm having to guess which box is which. 

As an alternative to colours if they are already used for variations, wonder about using solid/dashed/dotted lines as an alternative.

**Reply by NickM-27** [reply]
I think a general key would make more sense than trying to fit the explanation into a button color or pattern
